pyriemann/spatialfilters.py

```diff
@@ -467,3 +469,266 @@ def fit(self, X, y):
         self.patterns_ = A[:, 0:self.nfilter].T
 
         return self
+
+
+class AJDC(BaseEstimator, TransformerMixin):
+    """Implementation of the AJDC.
+
+    The approximate joint diagonalization of Fourier cospectral matrices (AJDC)
+    [1] is a versatile tool for blind source separation (BSS) tasks based on
+    Second-Order Statistics (SOS), estimating spectrally uncorrelated sources.
+
+    It can be apply:
+        - on a single subject, to solve the classical BSS problem [1],
+        - on several subjects, to solve the group BSS (gBSS) problem [2],
+        - on several experimental conditions (for eg, baseline versus task).
+
+    AJDC estimates Fourier cospectral matrices by the Welch's method, and
+    applies a trace-normalization. If necessary, it averages cospectra across
+    subjects, and concatenates them along experimental conditions.
+    Then, a dimension reduction and a whitening are applied on cospectra.
+    An approximate joint diagonalization (AJD) [3] allows to estimate the joint
+    diagonalizer, not constrained to be orthogonal. Finally, forward and
+    backward spatial filters are computed.
+
+    Parameters
+    ----------
+    window : int (default 128)
+        The length of the FFT window used for spectral estimation.
+    overlap : float (default 0.5)
+        The percentage of overlap between window.
+    fmin : float | None , (default None)
+        The minimal frequency to be returned. Since BSS models assume zero-mean
+        processes, the first cospectrum (0 Hz) must be excluded.
+    fmax : float | None , (default None)
+        The maximal frequency to be returned.
+    fs : float | None , (default None)
+        The sampling frequency of the signal.
+    expl_var : float (default 0.999)
+        The percentage of explained variance in ]0, 1], for dimension reduction
```

Comment:
```suggestion
        The percentage of explained variance in (0, 1], for dimension reduction
```
let's us English notations https://en.wikipedia.org/wiki/Interval_(mathematics)